packages/shared/src/components/sidebar/ContributeSection.tsx

```diff
@@ -46,39 +31,38 @@ export function ContributeSection({
       ),
       title: 'Submit article',
       action: trackAndShowSubmitArticle,
-      active: showSubmitArticle,
+      active: modal.type === LazyModal.SubmitArticle,
     },
     {
       icon: (active) => (
         <ListIcon Icon={() => <EmbedIcon secondary={active} />} />
       ),
       title: 'Suggest new source',
-      action: () => setShowNewSourceModal(true),
-      active: showNewSourceModal,
+      action: () => openModal({ type: LazyModal.NewSource }),
+      active: modal.type === LazyModal.NewSource,
     },
   ];
 
+  useEffect(() => {
+    const search = new URLSearchParams(window.location.search);
+    const query = Object.fromEntries(search);
+    if (!query?.scout) {
+      return;
+    }
```

Comment:
Where was this handled before, is there still a case for it?

Reply:
Relevant comment: https://github.com/dailydotdev/apps/pull/2160#discussion_r1321037239